documents/src/pages/build-element/events.md

```diff
@@ -1,38 +1,38 @@
 <!--
 title: Events
-location: ./events
+location: ./custom-components/events
 type: page
 layout: default
 -->
 
-::status-complete::
+
 
 # Events
 
-Events allow developers to know what has been changed, either by a user, or asynchronously.
+Events allow developers to know what has been changed on a component, either by a user, or asynchronously.
 
-## Inbuilt Events
+## Built-In Events
 
-There are two types of inbuilt events to use. These help with compatibility across multiple browsers and devices, and should be used instead of their native counterparts.
+Built-In events are events that comes with EF base class. There are two types of the events to use. These help with compatibility across multiple browsers and devices, and should be used instead of their native counterparts.
```

Comment:
```suggestion
Built-In events are events that comes with EF base class. There are two types of the events to use. These help with compatibility across multiple browsers and devices. It should be used instead of their native counterparts.
```